include/maidsafe/routing/types.h

```diff
@@ -85,6 +85,9 @@ using HandleGetReturn =
     boost::expected<boost::variant<std::vector<DestinationAddress>, std::vector<byte>>,
                     maidsafe_error>;
 using HandlePutPostReturn = boost::expected<std::vector<DestinationAddress>, maidsafe_error>;
+using HandlePostReturn =
```

Comment:
Probably a good candidate for the "Blocker" category in Jira.  I agree that it's usually bad to do more than the task says you should, but on the other hand we have a bad habit of leaving todo's in the code forever.  So I think this *should* be changed as part of the original task, but it wasn't anticipated when the task was written.